pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -266,8 +315,11 @@ func (r *ReconcileScanSettingBinding) Reconcile(ctx context.Context, request rec
 	}
 
 	if scanSettingBindingStatusNeedsUpdate(instance) {
+		found := compliancev1alpha1.ComplianceSuite{}
+		err = r.Client.Get(context.TODO(), types.NamespacedName{Namespace: suite.Namespace, Name: suite.Name}, &found)
```

Comment:
Should do some error handling here.

I had to refetch the ComplianceSuite, otherwise we were updating a stale reference.